xattr_freebsd.go

```diff
@@ -30,6 +35,25 @@ func getxattr(path string, name string, data []byte) (int, error) {
 	return int(r0), nil
 }
 
+func lgetxattr(path string, name string, data []byte) (int, error) {
```

Comment:
Here we can call one function (like on _linux_) and just pass `syscall.SYS_EXTATTR_GET_FILE` or `syscall.SYS_EXTATTR_GET_LINK`